cli-plugins/apps/create-workload.hbs.md

```diff
@@ -57,26 +57,15 @@ Tanzu Application Platform supports creating a workload from an existing local p
     - `--source-image` is the registry path where the local source code will be uploaded as an image.
 
     **Exclude Files**
-    When working with local source code, you can exclude files from the source code to be uploaded within the image by creating a file `.tanzuignore` at the root of the source code. You can find the options available to specify the workload in the command reference for [`workload create`](command-reference/tanzu-apps-workload-create.md), or run `tanzu apps workload create --help`.
+    When working with local source code, you can exclude files from the source code to be uploaded within the image by creating a file `.tanzuignore` at the root of the source code.
```

Comment:
When working with local source code, you can exclude files from the source code to be uploaded within the image by creating a `.tanzuignore` file at the root of the source code.